cmd/exchaind/main.go

```diff
@@ -47,6 +48,9 @@ func main() {
 	cobra.EnableCommandSorting = false
 
 	cdc := codec.MakeCodec(app.ModuleBasics)
+	reg := codec.MakeIBC(app.ModuleBasics)
+	proC := proxyCodec.NewProtoCodec(reg)
+	proxy := proxyCodec.NewCodecProxy(proC, cdc)
```

Comment:
I think these line code may convert to one function.

like:
func MakeProxy(module ModuleBasics) CodecProxy
{
   cdc := codec.MakeCodec(module)
	reg := codec.MakeIBC(module)
	proC := proxyCodec.NewProtoCodec(reg)
	proxy := proxyCodec.NewCodecProxy(proC, cdc)
       return proxy
}

thus is much better